test/bridge.jl

```diff
@@ -160,6 +160,32 @@ MOIU.@model(NoRSOCModel,
         @test bridgedmock.dist[(MathOptInterface.VectorOfVariables, MathOptInterface.RotatedSecondOrderCone)] == 1
     end
 
+    @testset "Supports" begin
+        fullbridgedmock = MOIB.fullbridgeoptimizer(mock, Float64)
+        for F in [MOI.SingleVariable, MOI.ScalarAffineFunction{Float64},
+                  MOI.ScalarQuadraticFunction{Float64}]
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.Interval{Float64})
+        end
+        for F in [MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64},
+                  MOI.VectorQuadraticFunction{Float64}]
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.PositiveSemidefiniteConeSquare)
+            @test MOI.supports_constraint(fullbridgedmock, F,
+                                          MOI.GeometricMeanCone)
+        end
+        for F in [MOI.VectorOfVariables, MOI.VectorAffineFunction{Float64}]
+            # TODO Missing vcat for quadratic for supporting quadratic
```

Comment:
I still don't understand these TODOs and how they relate to the surrounding code. Are they implying that certain constraints are not yet supported? Which?

Reply:
These tests are not in the previous `for` because they do not support VectorQuadraticFunction yet

Reply:
Could you clarify that on the comments?